Classes/Controller/VarnishCacheController.php

```diff
@@ -62,6 +31,18 @@ class VarnishCacheController extends AbstractModuleController
         'json' => JsonView::class,
     ];
 
+    #[Flow\Inject]
+    protected SiteRepository $siteRepository;
+
+    #[Flow\Inject]
+    protected ContentRepositoryRegistry $contentRepositoryRegistry;
+
+    #[Flow\Inject]
+    protected SiteNodeUtility $siteNodeUtility;
+
+    #[Flow\Inject]
+    protected ContentCacheFlusherService $contentCacheFlusherService;
```

Comment:
same here: lazy => eager

Reply:
Ok, didn't know that. Any way to use PHP Attributes and lazy injecton? Does it matter in these cases?

Reply:
No, I don't think that's possible.
And no, I don't think that it's a problem really, but I just wanted to mention it